Fix freeze attack checks after the introduction of workflow maximum duration (by removing workflow maximum duration?) (#136)
theupdateframework/specification: tuf-spec.md

  still be able to update again in the future. Errors raised during the update
  process should not leave clients in an unrecoverable state.

  **5.0**. **Record the time at which the update began** as the fixed update
  expiration time.  Time is fixed at the beginning of the update workflow to

>   allow an application using TUF to effectively pause time, in order to prevent

Author: You are absolutely correct. I'll replace with fixed update start time, this term is from an earlier attempt at defining this process. Well caught, thank you.

  metadata which is valid at the beginning of an update from expiring during

>   the update worfklow.

Reviewer: ```suggestion
  the update workflow.
```

>   allow an application using TUF to effectively pause time, in order to prevent
>   metadata which is valid at the beginning of an update from expiring during
>   the update workflow.

Reviewer 2: Similarly, I'm unsure about the causality implied in "pause time, in orderer to prevent". Don't we want to say something like this...?

```suggestion
  allows an application using TUF to effectively pause time, without 
  metadata, which is valid at the beginning of an update, becoming expired during
  the update workflow.
```

Author: How about:

> pause time, in order to ensure that metadata which has a valid expiration time at the beginning of the update does not fail an expiration check later in the update workflow

?

Reviewer 2: I just realized that I mistook "pause time" for "pause execution", so the original phrasing does make sense and so does your update. I withdraw my suggestion and am fine with both of yours.
